source/features/index.tsx

```diff
@@ -109,7 +109,7 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 		Object.assign(options, bisectedFeatures);
 	} else {
 		// If features are remotely marked as "seriously breaking" by the maintainers, disable them without having to wait for proper updates to propagate #3529
-		void updateHotfixes();
+		void updateHotfixes(version);
```

Comment:
?



The cache is already updated when the version changes, there's only ever one cache per version. This applies to both JS and CSS fixes

Reply:
We need it to compare to the unaffected versions when filtering. Also for `updateStyleHotfixes()` it's used to construct the file URL.

Though I guess it could be grabbed directly in `hotfix.ts`.